src/pkg/retention/job.go

```diff
@@ -110,6 +111,24 @@ func (pj *Job) Run(ctx job.Context, params job.Parameters) error {
 
 	// Log stage: results with table view
 	logResults(myLogger, allCandidates, results)
+	var delNum int
```

Comment:
actually, those codes can be put into 1 local function.

Reply:
Actually I don't like to use one string to store two integer, and UI may have to decode and validate them.